newton/solvers/solver_mujoco.py

```diff
@@ -1163,6 +1314,7 @@ def add_geoms(warp_body_id: int, perm_position=False, incoming_xform=None):
                 mass=body_mass[child],
                 ipos=pos2mjc(body_com[child, :]),
                 fullinertia=[inertia[0, 0], inertia[1, 1], inertia[2, 2], inertia[0, 1], inertia[0, 2], inertia[1, 2]],
+                explicitinertial=True,
```

Comment:
This is great! Thanks for fixing this, this should write the inertia to the MJCF hopefully. Although we might have to double-check the frame / world up axis handling here potentially.

Let's link this to https://github.com/newton-physics/newton/issues/125